### ISSUE TITLE
Strange decision to remove dbname from connection string in dbCreator initialization routine

### ISSUE THREAD

**Issue by maksm90** [timescale/tsbs#39]
I have noticed that `tsbs_load_timescaledb` cannot connect to database using specific username and dbname distinct from this name, e.g., using the following connection string `dbname=postgres user=user`. The problem lies in the following fragment of code _cmd/tsbs_load_timescaledb/creator.go:22_ :
```golang
// Needed to connect to user's database in order to drop/create db-name database
re := regexp.MustCompile(`(dbname)=\S*\b`)
d.connStr = strings.TrimSpace(re.ReplaceAllString(d.connStr, ""))
```

AFAIU the decision to remove dbname from connection string was based on assumption to distinguish the dbname for connection from the working dbname for benchmark. Instead of deletion I propose to check that these names obtained from connection string and option `-db-name` correspondingly are different.

**Comment by RobAtticus** [comment]
That makes sense, we can definitely add a flag like `-conn-db` to specify the database name to connect to for original creation. Did not think of that use case, but good point.

**Comment by antekresic** [comment]
I believe we can close this one since it was resolved by PR #60.

**Comment by maksm90** [comment]
Ok, thanks
